cli/types/cypress.d.ts

```diff
@@ -5479,6 +5490,13 @@ declare namespace Cypress {
      * @see https://on.cypress.io/catalog-of-events#App-Events
      */
     (action: 'window:load', fn: (win: AUTWindow) => void): Cypress
+    /**
+     * Fires when 'window:load' fires, as well as when a page on a secondary
+     * domain loads.
+     *
+     * WARNING: Intended for internal use. Subject to change or be removed.
+     */
+    (action: 'internal:window:load', fn: (details: InternalWindowLoadDetails) => void): Cypress
```

Comment:
Can we extend the interface internally where we need to with interface merging and not expose this here?

Reply:
Yes indeed, good idea. Addressed in [`157ea09`](https://github.com/cypress-io/cypress/pull/19430/commits/157ea0907a61600f33fef7a0ced09f80e0fd479f).